frontend/Themes.jsx

```diff
@@ -136,7 +160,7 @@ const StartView = styled.View`
 `;
 
 export {
-  BodyText, ButtonText, CardTitle, CardView, DataView, DataPointView,
-  DataScrollView, DetailedSettingsView, PageTitle, PageView, PressableButton,
-  SettingsView, Subheading, StartView,
+  BodyText, ButtonText, CardTitle, CardText, CardView, DataView, DataPointView,
+  DataScrollView, DetailedSettingsView, HomeCardData, HomeCardData2, HomeCardTitle, HomeCardView,
+  PageTitle, PageView, PressableButton, Subheading, StartView,
 };
```

Comment:
It might be cleaner if you export your styled components as you define them, instead of all at the end. It will make it easier for others to quickly see if a particular component is exported or if it's only used internally, instead of having to parse the export statement at the bottom of the file.

See #62